src/ui/src/abstractlogview.cpp

```diff
@@ -1047,7 +1047,7 @@ void AbstractLogView::wheelEvent( QWheelEvent* wheelEvent )
             // First see if we need to block the elastic (on Mac)
             if ( wheelEvent->phase() == Qt::ScrollBegin )
                 followElasticHook_.hold();
-            else if ( wheelEvent->phase() == Qt::ScrollEnd )
+            else if ( wheelEvent->phase() == Qt::ScrollEnd || wheelEvent->phase() == Qt::ScrollMomentum )
```

Comment:
The root cause is that there is no `Qt::ScrollEnd` phase on Mac, so the `followElasticHook_` will not decrease automatically.